plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergUtil.java

```diff
@@ -623,4 +623,11 @@ private static void validateOrcBloomFilterColumns(ConnectorTableMetadata tableMe
             throw new TrinoException(INVALID_TABLE_PROPERTY, format("Orc bloom filter columns %s not present in schema", Sets.difference(ImmutableSet.copyOf(orcBloomFilterColumns), allColumns)));
         }
     }
+
+    public static String removeRedundantSlash(String location)
+    {
+        // Version 393 and 394 stored metadata location with double slash. e.g. s3://bucket/db/table//metadata/00001.metadata.json
```

Comment:
Can you link the PRs that added this issue in 393 and which one fixed it in 395?

Reply:
Updated the code comment.